env.example

```diff
@@ -1,6 +1,21 @@
-export LOCATION=eastus
+export LOCATION=westeurope
 export ARO_IMAGE=arointsvc.azurecr.io/aro:latest
 export NO_CACHE=false
 export AZURE_EXTENSION_DEV_SOURCES="$(pwd)/python"
 
+# Change this to your preferred OpenShift version
+export OPENSHIFT_VERSION="4.13.40"
+
+export CLUSTER_RESOURCEGROUP="${USER}-v4-$LOCATION"
+export CLUSTER_NAME="${USER}-aro-cluster"
+export CLUSTER_VNET="${USER}-aro-vnet"
+
+# Check pkg/util/version/const.go for the correct values for OpenShift versions.
+# Set the image pullspecs for the chosen OpenShift version
+# Visit https://quay.io/repository/openshift-release-dev/ocp-release and filter the tags to find the correct release and SHA256.
+export OCP_PULLSPEC="quay.io/openshift-release-dev/ocp-release@sha256:<sha256>"
+
+# Update this with the correct installer pullspec for the chosen OpenShift version, for example (release-4.13)
+export INSTALLER_PULLSPEC="arointsvc.azurecr.io/installer:release-4.13"
```

Comment:
I don't think we should add these by default to the env, since using the default is normally the correct choice, unless we are updating the installer version.

Reply:
Fixed: in setup_resources.sh Fetch `OCP_PULLSPEC` dynamically

- Now we're dynamically fetching `OCP_PULLSPEC` with the right SHA256 for the OpenShift version.
- Also removed those default pullspec values from the environment variables to keep things clean and correct.